test/unit/lib/migration-payloads/validation/validateMigration-helper.js

```diff
@@ -0,0 +1,15 @@
+const Bluebird = require('bluebird');
```

Comment:
The name of this module is a it odd :)
Maybe let's call it `validate-payloads.js` or something like that.
I think we could leave out the `-helper` part since all our actual tests are using `.spec.js` so there shouldn't be much confusion.